pcsx2/SaveState.h

```diff
@@ -35,7 +35,7 @@ namespace Exception
 {
 	// ---------------------------------------------------------------------------------------
 	// Savestate Exceptions:
-	//   UnsupportedStateVersion / StateCrcMismatch
+	//   UnsupportedStateVersion
```

Comment:
`UnsupportedStateVersion` also seems to be unused. Maybe remove it as well?

Reply:
Looks like you're right, changes pushed.